higher_health/views.py

```diff
@@ -2,16 +2,17 @@
 from django.shortcuts import render
 
 from .forms import HealthCheckLogin, HealthCheckQuestionnaire
+from .utils import get_risk_level
 
 
 def healthcheck_questionnaire(request):
     submitted = False
     if request.method == "POST":
         form = HealthCheckQuestionnaire(request.POST)
         if form.is_valid():
-            # cd = form.cleaned_data
-            # assert False
-            return HttpResponseRedirect("/?submitted=True")
+            cd = form.cleaned_data
+            risk_level = get_risk_level(cd)
+            return HttpResponseRedirect(f"/?submitted=True&risk_level={risk_level}")
     else:
         form = HealthCheckQuestionnaire()
         if "submitted" in request.GET:
```

Comment:
@erikh360 Actually, this is the problem 🙂-- this allows the form to accept a GET request.